GeoCASe_functions.py

```diff
@@ -75,3 +78,79 @@ def gather_data() -> dict:
         i += 1
 
     return geocase_data
+
+
+def gather_publishers() -> dict:
```

Comment:
Long function, try to split it up in multiple functions

Reply:
The functions can be even smaller, think 10-15 lines is the size to aim for. But this is already much better.